test/functional/fixtures/regression/gh-2846/testcafe-fixtures/index.js

```diff
@@ -2,4 +2,7 @@ fixture `GH-2846`;
 
 test(`Debug`, async t => {
     await t.debug();
+
+    // NOTE: for https://github.com/DevExpress/testcafe/issues/6605
+    await t.wait(1);
```

Comment:
Need to check that the test report contains only one `Using t.debug in headless mode` warning.

Reply:
The test already has this. Take a look here:
https://github.com/DevExpress/testcafe/blob/master/test/functional/fixtures/regression/gh-2846/test.js#L13

Reply:
Ok